.github/workflows/test.yml

```diff
@@ -10,37 +10,42 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.7", "3.9", "3.11"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
+        sqlalchemy-version: ["1.4", "2.0"]
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v4
-      with:
-        python-version: ${{ matrix.python-version }}
-        cache: pip
-        cache-dependency-path: pyproject.toml
-    - name: Install dependencies
-      run: pip install -e .[test] coveralls
-    - name: Test with pytest
-      run: coverage run -m pytest
-    - name: Upload Coverage
-      run: coveralls --service=github
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        COVERALLS_FLAG_NAME: ${{ matrix.test-name }}
-        COVERALLS_PARALLEL: true
-        SQLALCHEMY_WARN_20: "true"
+      - uses: actions/checkout@v3
```

Comment:
I only just noticed: what caused the reformatting? That makes it hard to see the actual changes.

Reply:
If you could undo the formatting changes and only leave the actual changes in, then I'm ready to merge.

Reply:
Formatting removed in #275